rotkehlchen/api/v1/schemas.py

```diff
@@ -2869,13 +2873,17 @@ def validate_schema(
             data: dict[str, Any],
             **_kwargs: Any,
     ) -> None:
-        if data['identifier'] == 1 and data['name'] != ETHEREUM_ETHERSCAN_NODE_NAME:
+        etherscan_node_name = self.dbhandler.check_if_is_etherscan_node(data['identifier'])
+        if etherscan_node_name is not None and data['name'] != etherscan_node_name:
```

Comment:
here becomes difficult to read due the name. You expect it to be a boolean the output but is `None` or string.

Why not make a function that returns a set of the etherscan names (query by lengh in sqlite or in python) and 

```suggestion
        etherscan_rpc_names = self.dbhandler.get_etherscan_rpc_names(data['identifier'])
        if etherscan_node_name is not None and data['name'] not in etherscan_rpc_names
```

Reply:
In this case `etherscan_rpc_names` will be a list/set with always at max 1 element. This looks even more confusing to me. I have some other idea in mind on how to make this more readable and I will try to implement it.